source/features/dim-bots.tsx

```diff
@@ -20,8 +20,10 @@ const commitSelectors = botNames.map(bot => `.commit-author[href$="?author=${bot
 commitSelectors.push('.commit-author[href$="%5Bbot%5D"]'); // Generic `[bot]` label in author name
 const commitSelector = commitSelectors.join(',');
 
-const prSelectors = botNames.map(bot => `.opened-by [title="pull requests opened by ${bot}"]`);
+const prSelectors = [];
 prSelectors.push(
+	...botNames.map(bot => `.opened-by [title*="pull requests opened by ${bot}"]`),
+	...botNames.map(bot => `.opened-by [title*="pull requests created by ${bot}"]`), // PRs in https://github.com/pulls page
 	'.opened-by [href*="author%3Aapp%2F"]', // Search query `is:pr+author:app/*`
 	'.labels [href$="label%3Abot"]', // PR tagged with `bot` label
 );
```

Comment:
```suggestion
const prSelectors = [
	...botNames.map(bot => `.opened-by [title*="pull requests opened by ${bot}"]`),
	...botNames.map(bot => `.opened-by [title*="pull requests created by ${bot}"]`), // PRs in https://github.com/pulls page
	'.opened-by [href*="author%3Aapp%2F"]', // Search query `is:pr+author:app/*`
	'.labels [href$="label%3Abot"]', // PR tagged with `bot` label
];
```

Reply:
🤦🏻  I was too happy to see it working... 🙃 it's fixed